OpenRA.Mods.Common/Widgets/Logic/MissionBrowserLogic.cs

```diff
@@ -190,7 +190,7 @@ void CreateMissionGroup(string title, IEnumerable<MapPreview> previews)
 				var item = ScrollItemWidget.Setup(template,
 					() => selectedMap != null && selectedMap.Uid == preview.Uid,
 					() => SelectMap(preview),
-					StartMissionClicked);
+					new Action(() => StartMissionClicked(onExit)));
```

Comment:
Could this be just `() => StartMissionClicked(onExit)`?

Reply:
Yes. 
For some reason that didn't work for me at first, but now it seems correct